java/api/src/main/java/io/ray/api/parallelactor/ParallelActorCreator.java

```diff
@@ -0,0 +1,34 @@
+package io.ray.api.parallelactor;
+
+import io.ray.api.Ray;
+import io.ray.api.function.RayFuncR;
+
+public class ParallelActorCreator<A> {
+
+  private ParallelStrategyInterface strategy = null;
+
+  private RayFuncR<A> func;
+
+  private Object[] args;
+
+  public ParallelActorCreator(RayFuncR<A> func, Object[] args) {
+    //    ObjectRef
+    this.func = func;
+    this.args = args;
+
+    // By default, it should be ALWAYS_FIRST with 1 parallel.
+    ParallelContext ctx = Ray.internal().getParallelContext();
+    this.strategy = ctx.createParallelStrategy(ParallelStrategy.ALWAYS_FIRST, 1);
+  }
+
+  public ParallelActorCreator<A> setParallels(ParallelStrategy strategy, int parallelNum) {
```

Comment:
1. We shouldn't use an enum `ParallelStrategy` as the parameter. Because then it's not possible for users to write their own strategy. Why not just use `ParallelStrategyInterface` as the parameter? 
2. better separate this into 2 methods `setStrategy` and `setParallelism`

Reply:
I updated to
```java
Parallel.actor(A::new).setStrategy(new RoundRobinStrategy(/*parallelNum=*/10)).remote();
```

Reply:
@jovany-wang I don't think it's reasonable to put the parallelism info into the strategy class. I think `Parallel.actor(A::new).setParallelism(10).setStrategy(new RoundRobinStrategy()).remote()` is much better, because the parallelism option should not be a property of a strategy. And in this case, a user can omit `setStrategy` if one wants to use the default strategy. Still, `setParallelism` should be mandatory because the default value of 1 doesn't make sense.

Reply:
All stuffs about strategy were removed.